coderd/fileszip_test.go

```diff
@@ -0,0 +1,217 @@
+package coderd_test
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"bytes"
+	"io"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/coderd"
+	"github.com/coder/coder/v2/testutil"
+)
+
+// refTime is the Go reference time. The contents of the sample tar and zip files
+// in testdata/ all have their modtimes set to the below in some timezone.
+var refTime = time.Date(2006, 1, 2, 3, 4, 5, 0, must(time.LoadLocation("MST")))
```

Comment:
Nit: Ambiguous variable name in huge package, perhaps `archiveRefTime`?

Reply:
updated